pkgs/applications/networking/cluster/tftui/default.nix

```diff
@@ -0,0 +1,53 @@
+{ lib
+, buildPythonApplication
+, fetchPypi
+, makeWrapper
+, poetry-core
+, posthog
+, pyperclip
+, requests
+, rich
+, textual
+, enableUsageTracking ? false
+}:
+
+buildPythonApplication rec {
+  pname = "tftui";
+  version = "0.12.4";
+  format = "pyproject";
```

Comment:
Should be `pyproject = true;` now (see https://github.com/NixOS/nixpkgs/commit/39c64e232328e09284c766d2b56f2d680e511d41).

Reply:
Thanks for the tip, fixed in 795f3e1cbaae